### DISCUSSION TITLE
Rockchip-Powered Network Solutions | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1251] [Q&A]
# Rockchip-Powered Network Solutions | Seeed Studio Wiki

Rockchip-Powered Network Solutions

https://wiki.seeedstudio.com/Rockchip_network_solutions/

**Comment by Karavanka** [comment]
Hi there, 

Are these pocket routers equipped with OpenWRT or something similar? Could they be used as a VPN server or client? 
Thanks,
Ivan

**Reply by zogness** [reply]
They come with a fork (version) of OpenWrt installed. I think it's called immortalwrt. These devices are not officially supported by OpenWrt so there is probably no image file on the openwrt.org website you can download and install for LinkStar H86K products.
